package.yaml

```diff
@@ -1,5 +1,5 @@
 name: stackctl
-version: 1.6.1.2
```

Comment:
> This changes the interface of Generate to deal in ParametersYaml
values instead of (Amazonka) Parameters, so this will need a major
version bump.

Is the most significant digit not the major one?

Reply:
It's confusing. Haskell has its own versioning policy known as PVP, which is 4 part, considering the first two parts together as MAJOR. But we generally like to follow Semantic Version, which is 3 part. So in our Haskell packages we consider the version string to be `EPOCH.MAJOR.MINOR.PATCH`. We follow SemVer in the last 3 components then reserve EPIC for like total rewrites or major overhauls (like PlatformCLI v3).

I probably should've just used 3-part SemVer for Stackctl and PlatformCLI, since they're just tools and not libraries playing in the Haskell ecosystem, but mistakes were made.